src/main/scala/me/jeffmay/neo4j/client/Config.scala

```diff
@@ -0,0 +1,76 @@
+package me.jeffmay.neo4j.client
+
+import com.typesafe.config.Config
+
+import scala.concurrent.duration._
+import scala.util.Try
+
+case class Neo4jBasicAuth(username: String, password: String)
+object Neo4jBasicAuth {
+
+  /**
+    * The initial username and password of Neo4j that you are asked to change; just for reference.
+    */
+  final val Initial: Neo4jBasicAuth = Neo4jBasicAuth("neo4j", "neo4j")
+
+  /**
+    * You should really change these, but for testing purposes these are useful defaults.
+    */
+  final val UnsafeDefaultPassword: Neo4jBasicAuth = Neo4jBasicAuth("neo4j", "password")
+
+  def fromConfig(config: Config, path: String = Neo4jClientConfig.DefaultRoot): Neo4jBasicAuth = {
+    // TODO: Handle nulls
+    val c = config.getConfig(path)
+    Neo4jBasicAuth(
+      username = c.getString("username"),
+      password = c.getString("password")
+    )
+  }
+}
+
+case class Neo4jClientConfig(
+  baseUrl: String,
+  timeout: FiniteDuration = 2.seconds,
+  autoInitialize: Boolean = true,
+  credentials: Neo4jBasicAuth = Neo4jBasicAuth.Initial,
+  includeStatsByDefault: Boolean = false
+)
+
+object Neo4jClientConfig {
+
+  /**
+    * The default root of the config file from which to read configs for this library.
+    *
+    * This default assumes a fairly flat config file. You may want to consider nesting
+    * this config under some deeper nested block. Providing your custom root to any
+    * of the config parsers will insure you don't get runtime errors.
+    */
+  final val DefaultRoot: String = "neo4j"
+
+  /**
+    * Use localhost default settings for the database.
+    */
+  final val Local: Neo4jClientConfig = Neo4jClientConfig("http://localhost:7474")
+
+  /**
+    *
+    * @param config the config file or sub-object
+    * @param path the root path in the config file from which all configs in this library are read.
+    * @return
+    */
```

Comment:
Cleanup. Half-comments or erroneous comments are worse than no comments at this point.